src/ts/components/subtitleoverlay.ts

```diff
@@ -4,30 +4,114 @@ import SubtitleCueEvent = bitmovin.PlayerAPI.SubtitleCueEvent;
 import {Label, LabelConfig} from './label';
 import {ComponentConfig, Component} from './component';
 import {ControlBar} from './controlbar';
+import {ColorUtils, StorageUtils} from '../utils';
+import {DOM} from '../dom';
+
+export interface SubtitleOverlayConfig extends ContainerConfig {
```

Comment:
Simplify to 

```ts
export interface SubtitleOverlayConfig extends ContainerConfig {
  subtitleConfig?: UISubtitleConfig;
}
```

The `subtitleConfig` is optional. If specified, it takes precedence over the global UI config from `UIManager#getConfig()`